MainRepository/Program.cs

```diff
@@ -1,7 +1,5 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿internal class Program {
+   private static void Main (string[] args) {
+      TStack<int> stack = new ();

```

Comment:
Why? Either you should have used this stack and implement few actions or this should not have been added.